modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -0,0 +1,16 @@
+[[tuning-changelogs]]
+= Tuning Changelogs
+
+Some packages have a long list of changelog entries.
+This data is downloaded by default, but is very rarely useful information.
```

Comment:
We can keep this even when it is not really correct. changelog info are useful. But most of the tools used for installing updates cannot show them. They download data which are not used.

In addition the question is, if 10 years old changelog entries are of any interest when you want to patch a system.